src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -234,9 +234,19 @@ public Task CallFunctionAsync(string functionName, params object[] parameters)
 
             // TODO: Support for versioning
             string version = DefaultVersion;
-            this.config.AssertActivityExists(functionName, version);
+            FunctionType functionType = this.config.GetFunctionType(functionName, version);
```

Comment:
>FunctionType [](start = 12, length = 12)

Rather than introducing a new enum, I would suggest exposing methods like TryGetActivityFunction and TryGetOrchestratorFunction.  It's slightly more performant and introduces fewer new concepts.